google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/facet/StackdriverFrameworkType.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.debugger.facet;
+
+import com.google.cloud.tools.intellij.ui.GoogleCloudToolsIcons;
+import com.google.cloud.tools.intellij.util.GctBundle;
+
+import com.intellij.framework.FrameworkTypeEx;
+import com.intellij.framework.addSupport.FrameworkSupportInModuleProvider;
+
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.Icon;
+
+public class StackdriverFrameworkType extends FrameworkTypeEx {
```

Comment:
Can you explain why we need a framework type for Stackdriver? It was my understanding that the facet will hook onto the App Engine Standard framework initialization.